README.md

```diff
@@ -5,16 +5,41 @@
 This repo is a port from the original `JPSreport` to a Python implementation, and will provide the same functionalities.
 
 ## Getting started
-### Setup dependencies
+### Setup Python
 For setting up your Python Environment a Python version >= 3.8 is recommended (our code is tested with 3.8).
 To avoid conflicts with other libraries/applications the usage of virtual environments is recommended, see [Python Documentation](https://docs.python.org/3/library/venv.html) for more detail.
 
-Now the needed libraries for `PiTA` need to be installed.
-You can use the provided `requirements.txt` to do so:
 
+### Installing PiTA
+You can install the latest version of PiTA from the test-Python Package 
+index via
 ```bash
-pip install -r requirements.txt
+python3 -m pip install --pre --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ pitapy
 ```
+This will also install all the dependencies for using the library directly.
 
 ### Usage
 The usage of the current state of the porting process can be seen in [demo notebook](usage.ipynb).
+You can either download the notebook and [demo files](demos/bottleneck/) 
```

Comment:
I would remove all this download-stuff. A direct link to the notebook is enough IMHO. 
Otherwise, some might think they really need to download everything just to use the notebook.

Reply:
Without mentioning that the demo files need to be downloaded, people may wonder why the notebook doesn't work for them.